source/Calamari.Azure.Tests/Fixtures/Conventions/DeployAzureResourceGroupConventionFixture.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Calamari.Azure.Deployment.Conventions;
+using Calamari.Azure.Deployment.Integration.ResourceGroups;
+using Calamari.Integration.FileSystem;
+using NUnit.Framework;
+
+namespace Calamari.Azure.Tests.Fixtures.Conventions
+{
+    public class GenerateDeploymentNameFromStepNameTestWrapper : DeployAzureResourceGroupConvention
```

Comment:
Can we extract the code that generates the deployment name to a separate static function and test is directly? In that case we would not need the additional wrapper and reduce the noise